### PR TITLE
Add HomebrewStore

### PR DESCRIPTION
**Pull request by DerfJagged**
A script that allows you to download homebrew games, apps, dashboards, and emulators straight to your console.

### CODE REVIEW

File: UtilityScripts/HomebrewStore/Main.lua [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+	::mainend::
+	FileSystem.DeleteDirectory(absoluteDownloadsPath);
+	print("-- " .. scriptTitle .. " Ended...");
+	::scriptend::
+end
+
+function init()
+	-- Clear out unfinished downloads
+	downloadsPath = "Downloads\\";
+	absoluteDownloadsPath = Script.GetBasePath() .. downloadsPath;
```
Comment (Swizzy) [inline UtilityScripts/HomebrewStore/Main.lua:43]: This should probably be defined on line 11 rather than here and then not changed

File: UtilityScripts/XefuSpoofer/Main.lua [hunk 2]
```diff
@@ -1,6 +1,6 @@
 scriptTitle = "Xefu Spoofer"
 scriptAuthor = "Derf"
-scriptVersion = 1.1
+scriptVersion = 2
```
Comment (Swizzy) [inline UtilityScripts/XefuSpoofer/Main.lua:3]: Why is this change included here? it has nothing to do with the HomebrewStore.

Comment (DerfJagged) [reply UtilityScripts/XefuSpoofer/Main.lua:3]: Didn't realize me pushing the change would go into this PR. The fix was just because Aurora displays 1.1 as just "1" on the console itself, so it needs to be changed to "2" to tell it apart from the original release.

File: UtilityScripts/HomebrewStore/Main.lua (hunk 1 above)
Comment (DerfJagged) [reply UtilityScripts/HomebrewStore/Main.lua]: Done.

Comment (DerfJagged) [reply UtilityScripts/HomebrewStore/Main.lua:43]: It appears that "Script" cannot be called from outside of a function. I've moved the second line back to where it was originally.

File: UtilityScripts/HomebrewStore/Main.lua [hunk 3]
```diff
@@ -0,0 +1,457 @@
...
+		else
+			Script.ShowMessageBox("ERROR", "An unknown error occurred!\n\nExiting...", "OK");
+		end
+	end
+end
+
+function HandleSelection(selection, repo, menu)
+	local info = "";
+	info = info .. "Name: " .. selection.itemTitle .. "\n";
+	if selection.itemDescription ~= nil and selection.itemDescription ~= "" then
```
Comment (Swizzy) [inline UtilityScripts/HomebrewStore/Main.lua:172]:
```suggestion
	if selection.itemVersion ~= nil and selection.itemVersion ~= "" then
```
